pkg/didmethod/peer/store.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/storage"
 )
 
+const (
+	// StoreNameSpace store name space for DID Store
+	StoreNameSpace = "didresolver"
```

Comment:
Namespace is one word.

Reply:
Does this need to be exported ?

Reply:
Yes, it is being used by default framework init.
Like protocols exporting their namespaces.

Reply:
done